cover.out

```diff
@@ -0,0 +1,43 @@
+mode: set
```

Comment:
did you mean to include this in your commit?

Reply:
Whoops - left that after checking to address the coverage comments